libs/react-helpers/src/lib/assets.ts

```diff
@@ -1,3 +1,5 @@
+import { AssetStatus } from '@vegaprotocol/types';
```

Comment:
question (non blocking): Wondering if this should be moved to libs/assets?

Reply:
It certainly can be moved here 👍🏻 it would be more sane to have it here